File: 1168.py
n = int(input())
lista = [6,2,5,4,5,6,3,7,6]
for j in range(n):
    ans = 0
    num = int(input())
    while(num):
        ans += lista[(num%10)]
        num /= 10
    print(ans, "leds")

File: 1168.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    long long n, num, ans=0;
    cin >> n;
    int lista[10] = {6, 2, 5, 5, 4, 5, 6, 3, 7, 6};
    while (n--){
        cin >> num;
        while(num){
            ans += lista[(num%10)];
            num = num/10;
        }
        cout << ans << " leds\n";
        ans = 0;
    }
}